src/RileyMaze/userFunction.js

```diff
@@ -240,6 +232,22 @@ ${code}
             return;
         }
 
+        if (scope.hasOwnProperty('riley') && scope.riley.queue.length !== MAX_QUEUE_LEN) {
+            // Add lines for the function name
+            let lineNum = MAX_QUEUE_LEN + 2;
+            if (scope.riley.queue.length < MAX_QUEUE_LEN)
+                lineNum = scope.riley.queue.length + 1;
```

Comment:
IIRC in the discussion on Randy's spec, we landed on showing the error on the last line of the code view in the case of too few instructions.